runtime/themes/nightfox.toml

```diff
@@ -0,0 +1,182 @@
+"ui.cursorline.primary" = { bg = "bg3" } # The line of the primary cursor (if cursorline is enabled)
+# "ui.cursorline.secondary" = { } #	The lines of any other cursors (if cursorline is enabled)
+# "ui.cursorcolumn.primary" = { } #	The column of the primary cursor (if cursorcolumn is enabled)
+# "ui.cursorcolumn.secondary" = { } #	The columns of any other cursors (if cursorcolumn is enabled)
+
+"ui.linenr" = { fg = "fg3" } # Line numbers.
+"ui.linenr.selected" = { fg = "yellow", modifiers = ["bold"] } # Current line number.
+
+# "ui.virtual" = { } # Namespace for additions to the editing area.
+"ui.virtual.ruler" = { bg = "bg3" } # Vertical rulers (colored columns in editing area).
+"ui.virtual.whitespace" = { fg = "bg3" } # Whitespace markers in editing area.
+"ui.virtual.indent-guide" = { fg = "black" } # Vertical indent width guides
+
+"ui.statusline" = { fg = "fg2", bg = "bg0" } # Status line.
+"ui.statusline.inactive" = { fg = "fg3", bg = "bg0" } # Status line in unfocused windows.
+"ui.statusline.normal" = { bg = "blue", fg = "bg0", modifiers = ["bold"] } # Statusline mode during normal mode (only if editor.color-modes is enabled)
+"ui.statusline.insert" = { bg = "green", fg = "bg0", modifiers = ["bold"] } # Statusline mode during insert mode (only if editor.color-modes is enabled)
+"ui.statusline.select" = { bg = "magenta", fg = "bg0", modifiers = ["bold"] } # Statusline mode during select mode (only if editor.color-modes is enabled)
+
+"ui.help" = { bg = "sel0", fg = "fg1" } # Description box for commands.
+
+"ui.menu" = { bg = "sel0", fg = "fg1" } # Code and command completion menus.
+"ui.menu.selected" = { bg = "fg3" } # Selected autocomplete item.
+"ui.menu.scroll"	= { fg = "fg3" } # fg sets thumb color, bg sets track color of scrollbar.
+
+"ui.popup" = { bg = "bg0", fg = "fg1" } # Documentation popups (space-k).
+"ui.popup.info" = { bg = "sel0", fg = "fg1" } # Info popups box (space mode menu).
+
+"markup.raw" = { fg = "magenta" } # Code block in Markdown.
+"markup.raw.inline" = { fg = "orange" } # `Inline code block` in Markdown.
+"markup.heading" = { fg = "yellow", modifiers = ["bold"] }
+"markup.list" = { fg = "magenta", modifiers = ["bold"] }
+"markup.bold" = { fg = "orange", modifiers = ["bold"] }
+"markup.italic" = { fg = "pink" }
+"markup.link" = { fg = "yellow-bright", modifiers = ["bold"] }
+"markup.quote" = { fg = "blue" }
+
+
+# DIAGNOSTICS
+"warning" = { fg ="yellow", bg = "bg1" } # Diagnostics warning (gutter)
+"error" = { fg = "red", bg = "bg1" } # Diagnostics error (gutter)
+"info" = { fg = "blue", bg = "bg1" } # Diagnostics info (gutter)
+"hint" = { fg = "green", bg = "bg1" } # Diagnostics hint (gutter)
+"diagnostic" = { modifiers = ["underlined"] } # Diagnostics fallback style (editing area)
+"diagnostic.error" = { fg = "red" } #	Diagnostics error (editing area)
+"diagnostic.warning" = { fg = "yellow" } #	Diagnostics warning (editing area)
+"diagnostic.info" = { fg = "info" } #	Diagnostics info (editing area)
```

Comment:
it looks like this is not defined in the palette

Reply:
good eye. fixed it. sorry for that.